src/v2/Apps/Conversation/Components/ConversationSnippet.tsx

```diff
@@ -88,7 +97,12 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
         underlineBehavior="none"
       >
         <Flex alignItems="center" px="5px" width="100%" height="120px">
-          <Flex alignItems="center" width="20px" height="100%">
+          <Flex
+            alignItems="center"
+            width={space(2)}
+            flexShrink="0"
```

Comment:
This `flex-shrink` makes the left side spacer unshrinkable so no matter what it will be `20px`.